cmd/stencil/update.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+// Description: This file contains code for the create command
+
+package main
+
+import (
+	"github.com/urfave/cli/v2"
+)
+
+// NewCreateCommand returns a new urfave/cli.Command for the
+// create command
+func NewUpdateCommand() *cli.Command {
+	return &cli.Command{
+		Name:        "update",
```

Comment:
I don't have a better idea, but `update` seems... a little too loose/generic here. What this command really does is configure a module. Maybe it should be `module configure`?